oscarapi/tests/testbasket.py

```diff
@@ -0,0 +1,939 @@
+import json
+from django.conf import settings
+from django.core.urlresolvers import reverse
+
+from oscar.core.loading import get_model
+
+from oscarapi.tests.utils import APITest
+
+
+Basket = get_model('basket', 'Basket')
+
+
+class BasketTest(APITest):
+    fixtures = [
+        'product', 'productcategory', 'productattribute', 'productclass',
+        'productattributevalue', 'category', 'attributeoptiongroup', 'attributeoption',
+        'stockrecord', 'partner'
+    ]
+
```

Comment:
It has several advantages. Sometimes you don't need a model in your database, so then using ``factory-boy`` is quicker. You're also more flexible, as you can change e.g. product names as suitable for your tests, and you can generate as many or as little as you need. Furthermore, explicitly creating just what you need is more explicit than relying on some catch-all collection of fixtures.

And, ahem, XML. 

But it's not a blocker; just something I'd ideally like to see improved.